### DISCUSSION TITLE
IR database from irplus

### DISCUSSION THREAD

**Discussion by yarbabin** [djsime1/awesome-flipperzero#30] [Ideas]
Hello.

There is database converted in Flipper format from irplus app https://play.google.com/store/apps/details?id=net.binarymode.android.irplus&hl=ru&gl=US

https://github.com/sasiplavnik/Flipper-IRDB

**Comment by geirskogul** [comment]
Love this!

**Comment by yoyo1306** [comment]
merci mec !

**Comment by UberGuidoZ** [comment]
Is this different than the existing [IRDB](https://github.com/UberGuidoZ/Flipper-IRDB)? If so, wouldn't it make the most sense to merge it in instead of having 2 different ones? Thank you for your work!

::EDIT:: This does appear to have some different vendors/models. I'd like to merge it into the main IRDB to keep everything in one place. (It currently gets around 35,000 visits A DAY with over half a million a week between the 3 main forks, all in a single line.) Obviously, proper credit will be given!

**Comment by djsime1** [comment]
Added.

**Comment by UberGuidoZ** [comment]
I'm merging this into the original IRDB to ensure it is seen by as many people as possible. Thank you yarabin!
